lib/rails_multitenant/version.rb

```diff
@@ -1,3 +1,3 @@
 module RailsMultitenant
-  VERSION = "0.1.0"
+  VERSION = '0.2.0'
```

Comment:
We already do since SemVer makes no API stability guarantees until you hit version 1.0 ;) That said this won't be very useful when this gem becomes more widespread (currently it's only used by the rules service and the Essendant review app). For now I've added a CHANGELOG.md to track user visible changes but once we have a few consumers happy with the gem we should aggressively promote it to 1.0.